src/modules/party/dto/get-parties-by-search-text-response.dto.ts

```diff
@@ -1,8 +1,8 @@
 import { ApiResponseProperty } from '@nestjs/swagger';
 
-export type GetPartiesBySearchTextResponse = GetPartiesBySearchTextResponseElement[];
+export type GetPartiesBySearchTextResponse = GetPartiesBySearchTextResponseItem[];
 
-export class GetPartiesBySearchTextResponseElement {
+export class GetPartiesBySearchTextResponseItem {
   @ApiResponseProperty()
```

Comment:
Should we also have `@Length()` decorator?

Reply:
This is a _response_ item rather than a request item - I believe we aren't doing any validation on responses?

Reply:
Yes! apologies please ignore my comment.